### PR TITLE
made changes to the performance calender

### LINKED ISSUE #3800: [UI] Improve calendar placement in performance page
#### Current Behavior
The placement of calendar is not very appealing. Some information is lost because of the calendar width being small.

Screenshot 2021-08-06 at 11 10 43 AM

Screenshot 2021-08-06 at 11 09 49 AM

#### Desired Behavior
The placement of calendar should be changed. 

#### Implementation
Move the calendar unto it's own card and make it span the entire width.

#### Acceptance Tests

- [ ] The card for `Service mesh metrics` should not be moved to the bottom.

#### Mockups

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Made changes to the calender.

There are 2 things that we could do:
1. if we make the cards the same size when the drawer is out, it becomes uneven when the drawer is in.

https://user-images.githubusercontent.com/53532328/128562341-7845b797-09cc-4900-ac7d-84b41a0e9cb4.mov

2. Or we could do it the other way around.

128489793-9cf29337-278d-4c54-a017-3310c8f676f0

128489217-008d01b9-4712-4bcc-9084-ace1d3900039

This PR fixes #3800 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
@sudo-NithishKarthik any pointers?

**Comment by sudo-NithishKarthik** [comment]
This will work @warunicorn19 👍🏽

File: ui/components/MesheryPerformance/Dashboard.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ function Dashboard({ updateProgress, enqueueSnackbar, closeSnackbar, grafana, ro
             <Grid container spacing={1}>
               <Grid item xs>
                 <Paper className={classes.paper}>
-                  <div style={{ display: "flex", alignItems: "center" }}>
+                  <div style={{ display: "flex", alignItems: "center" , height: "6.8rem"}}>
```
Comment (sudo-NithishKarthik) [inline ui/components/MesheryPerformance/Dashboard.js:115]: why are we using fixed height here ?

**Comment by warunicorn19** [comment]
Awesome!
FYI, ive done it the first way.

File: ui/components/MesheryPerformance/Dashboard.js (hunk 1 above)
Comment (warunicorn19) [reply ui/components/MesheryPerformance/Dashboard.js:115]: yes this is for the `service mesh metrics` card and the `profiles and results` card to be the same height

Comment (sudo-NithishKarthik) [reply ui/components/MesheryPerformance/Dashboard.js:115]: hmm ,actually, the `service mesh metrics` will expand when users have grafana or promethues configured, so, in that case, having a fixed height will not be desired

Comment (warunicorn19) [reply ui/components/MesheryPerformance/Dashboard.js:115]: oops, sorry my bad, the fixed heights are for the `results` and `profiles` card and not for the `service mesh metrics` , would that still be a problem?

Comment (sudo-NithishKarthik) [reply ui/components/MesheryPerformance/Dashboard.js:115]: Ok 👍🏽

File: ui/components/MesheryPerformance/Dashboard.js [hunk 2]
```diff
@@ -159,7 +158,13 @@ function Dashboard({ updateProgress, enqueueSnackbar, closeSnackbar, grafana, ro
           </Paper>
         </Grid>
       </Grid>
+      <Grid>
+        <Paper className={classes.paper}>
+      <PerformanceCalendar style={{ height: "40rem", margin: "2rem 0 0" }} />
```
Comment (sudo-NithishKarthik) [inline ui/components/MesheryPerformance/Dashboard.js:163]: This height will vary, so, better not to keep it fixed ✌🏽

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]

File: ui/components/MesheryPerformance/Dashboard.js (hunk 2 above)
Comment (warunicorn19) [reply ui/components/MesheryPerformance/Dashboard.js:163]: This was there from before itself, i didnt make any changes, if we dont specify a height the calendar collapses into a line.

Comment (sudo-NithishKarthik) [reply ui/components/MesheryPerformance/Dashboard.js:163]: Oh ,ok, so, instead of height, we can change it to `min-height` ig

Comment (warunicorn19) [reply ui/components/MesheryPerformance/Dashboard.js:163]: Ive tried doing that, but then the grid of the calendar goes away😬.

Comment (sudo-NithishKarthik) [reply ui/components/MesheryPerformance/Dashboard.js:163]: Ok, it will not create any issues as of now, so, we can stick with this 👍🏽

**Comment by sudo-NithishKarthik** [comment]
can you fix the linting issues @warunicorn19  ?

**Comment by warunicorn19** [comment]
i hope this fixes it.

**Review by leecalcote — APPROVED** [review]
Excellent.

**Review by leecalcote — APPROVED** [review]

**Review by sudo-NithishKarthik — APPROVED** [review]

**Pull request merged**